### ISSUE TITLE
Prediction 2 - Many replicated objects might be corrupting packet.

### ISSUE THREAD

**Issue by FirstGearGames** [FirstGearGames/FishNet#627]
FN Version 4.1.3-4.1.4

Many replicated objects (32-50) seem to prevent other objects from updating.

**Comment by FirstGearGames** [comment]
Attached project to replicate.
- Open P2 Large Reconcile Bug.
- Start server, join a client in another editor.
- Select Spawner on scene on server. Spawning roughly 25 objects or more, at once or overtime, results in reconcile datas not being sent for a number of objects.
[Missing Reconcile Data.zip](https://github.com/FirstGearGames/FishNet/files/14812799/Missing.Reconcile.Data.zip)

**Comment by FirstGearGames** [comment]
A solution has been found. It is temporary but will get the job done.

Open file NetworkConnection.Prediction.
Replace line...
```cs
if (writerCount == 0 || (data.Length + PredictionStateWriters[writerCount-1].Length) > mtu)
```
with this...
```cs
if (writerCount == 0)
```

**Comment by FirstGearGames** [comment]
Properly resolved next release 4.2.0!
